python/tutorials/06-fused-attention.py

```diff
@@ -80,7 +84,7 @@ def _attn_fwd_inner(acc, l_i, m_i, q,  #
         alpha = tl.math.exp2(m_i - m_ij)
         l_ij = tl.sum(p, 1)
         # -- update output accumulator --
-        if warp_specialize and BLOCK_M == 128 and HEAD_DIM == 128:
+        if warp_specialize and SUB_TILING and BLOCK_M == 128 and HEAD_DIM == 128:
```

Comment:
```suggestion
        if SUBTILE_EPILOGUE:
```

Since we're passing a flag for subtiling, let's control it from the launcher based on the same parameters.